CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Version 1.13.1
+## Version [1.13.1](https://github.com/jscs-dev/node-jscs/compare/v1.12.0...v1.13.1)
```

Comment:
Sorry wasn't sure what the link should be (the previous release or previous
minor release) but yeah I guess its not that useful
On Jul 12, 2015 9:59 PM, "Oleg Gaidarenko" <notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/jscs-dev/node-jscs/pull/1516#discussion_r34429358>:
>
> > @@ -1,4 +1,4 @@
> > -## Version 1.13.1
> > +## Version [1.13.1](https://github.com/jscs-dev/node-jscs/compare/v1.12.0...v1.13.1)
>
> Yep, feel free to fix this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jscs-dev/node-jscs/pull/1516/files#r34429358>.
>